.env

```diff
@@ -0,0 +1,2 @@
+WEATHER_API_USER=asekerich99
+WEATHER_API_PASSWORD=o3bG6WvKLzVI
```

Comment:
This is not something that anyone suppose to see. Look at .env conception. npm dotenv or webpack-dotenv